<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QKxMiniSessionItem</class>
 <widget class="QWidget" name="QKxMiniSessionItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>147</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>16</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="friendlyName">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QToolButton" name="btnClose">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="feidesk.qrc">
             <normaloff>:/resource/skin/close2.png</normaloff>:/resource/skin/close2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>8</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="active">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="btnDesktop">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="feidesk.qrc">
           <normaloff>:/resource/skin/mstsc2.png</normaloff>:/resource/skin/mstsc2.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnFtp">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="feidesk.qrc">
           <normaloff>:/resource/skin/ftp3.png</normaloff>:/resource/skin/ftp3.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnChat">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="feidesk.qrc">
           <normaloff>:/resource/skin/chat.png</normaloff>:/resource/skin/chat.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnTerminal">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="feidesk.qrc">
           <normaloff>:/resource/skin/terminal.png</normaloff>:/resource/skin/terminal.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnPrinter">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="feidesk.qrc">
           <normaloff>:/resource/skin/printer.png</normaloff>:/resource/skin/printer.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnCamera">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="feidesk.qrc">
           <normaloff>:/resource/skin/camera.png</normaloff>:/resource/skin/camera.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>6</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="feidesk.qrc"/>
 </resources>
 <connections/>
</ui>
